src/bluOs/player.ts

```diff
@@ -76,7 +92,7 @@ export const isSameTrack = (a: PlayingTrack, b: PlayingTrack): boolean =>
 
```

Comment:
Might want to update this, it is checking the title twice, and I believe this should also check the artist.

Reply:
Great catch, thanks! 😂

Reply:
Fixed.